src/ray/raylet/local_object_manager.h

```diff
@@ -68,12 +68,13 @@ class LocalObjectManager {
   /// Pin objects.
   ///
   /// \param object_ids The objects to be pinned.
-  /// \param objects Pointers to the objects to be pinned. The pointer should
-  /// be kept in scope until the object can be released.
   /// \param owner_address The owner of the objects to be pinned.
+  /// \param objects Pointers to the objects to be pinned. The pointer should
+  /// be kept in scope until the object can be released. If it's empty, it must
```

Comment:
Instead of `If it's empty, it must  /// have already been pinned.`,  why don't we comment `If it is empty it adds an owner address to the existing pinned object`?